plugins/modules/provision_workflow_manager.py

```diff
@@ -74,28 +74,41 @@
             type: list
             elements: str
         dynamic_interfaces:
-            description: Interface details of the controller
+            description: Interface details of the wireless controller
             type: list
             elements: dict
             suboptions:
+                interface_name:
+                    description: The name of the interface
+                    type: str
+                vlan_id:
+                    description: The VLAN ID associated with the interface.
+                    type: str
                 interface_ip_address:
-                    description: Ip Address allocated to the interface
+                    description: The IP address assigned to the interface.
                     type: str
-                interface_netmask_in_c_i_d_r:
-                    description: Ip Address allocated to the interface
-                    type: int
                 interface_gateway:
-                    description: Ip Address allocated to the interface
+                    description: The gateway IP address for the interface.
                     type: str
                 lag_or_port_number:
-                    description: Ip Address allocated to the interface
-                    type: int
-                vlan_id:
-                    description: Ip Address allocated to the interface
-                    type: int
-                interface_name:
-                    description: Ip Address allocated to the interface
+                    description: The port number or LAG (Link Aggregation Group) identifier.
                     type: str
+                interface_netmask_in_c_i_d_r:
+                    description: The netmask of the interface in CIDR format
+                    type: str
+        skip_ap_provision:
+            description: Skip AP provisioning
+            type: bool
+        rolling_ap_upgrade:
```

Comment:
Can you make it more clear? Check the below one and do some enhancements if required..

```
        rolling_ap_upgrade:
            description: 
              - Configuration options for performing a rolling upgrade of Access Points (APs) in phases.
              - Allows control over the gradual rebooting of APs during the upgrade process.
            type: dict
            suboptions:
                enable_rolling_ap_upgrade:
                    description: 
                      - Enable or disable the rolling AP upgrade feature.
                      - If set to 'true', APs will be upgraded in batches based on the specified reboot percentage.
                    type: bool
                    default: false
                ap_reboot_percentage:
                    description: 
                      - The percentage of APs to reboot simultaneously during an upgrade.
                      - Must be an integer value, typically between 1 and 100, indicating the proportion of APs to upgrade at a time.
                    type: int
                    default: 20

```

Reply:
Updated the code accordingly.